packages/cli/lib/sandboxes.js

```diff
@@ -66,25 +80,51 @@ async function getAvailableSyncTypes(parentAccountConfig, config) {
   return syncTypes.map(t => ({ type: t.name }));
 }
 
-const sandboxCreatePersonalAccessKeyFlow = async (env, account, name) => {
-  const configData = await personalAccessKeyPrompt({ env, account });
+/**
+ * @param {String} env - Environment (QA/Prod)
+ * @param {Object} result - Sandbox instance returned from API
+ * @param {Boolean} force - Force flag to skip prompt
+ * @returns {String} validName saved into config
+ */
+const saveSandboxToConfig = async (env, result, force = false) => {
+  // const configData = { env, personalAccessKey: result.personalAccessKey };
+  // TODO: Temporary, remove
+  const configData = await personalAccessKeyPrompt({
+    env,
+    account: result.sandbox.sandboxHubId,
+  });
+  // End temporary section
```

Comment:
NOTE: BE does not currently return PAK just yet, so we are keeping the old PAK prompt for now as a temporary measure.